src/sass/components/_global.scss

```diff
@@ -0,0 +1,5 @@
+html,
+body {
+  overflow-x: hidden;
```

Comment:
Nie za dobra praktyka. Jeśli coś dzieje się z naszą stroną i jest ona scrollowalna w poziomie to należy sprawdzić, który element to "psuje", ustawianie `overflow-x` na sztywno nie jest dobrą praktyką